installation/linux.md

```diff
@@ -646,6 +578,31 @@ sudo rm /lib/systemd/system/openhab2.service
 | Backups folder                   | `/var/lib/openhab2/backups`  | `/opt/openhab2/backups`           |
 | Service configuration            | `/etc/default/openhab2`      | (not preconfigured)               |
 
+## Backup and Restore
+
+It is recommended to make a backup of your configuration before *any* major change.
+To make a backup of openHAB2, you need to retain your configuration and userdata files.
+openHAB comes with scripts for storing your configuration in a zip file which is saved in `/var/lib/openhab2/backups` for automatic installs and `openhab2/backups` for manual installs.
+You can change the default path by setting the $OPENHAB_BACKUPS environment variable.
+
+```shell
+sudo $OPENHAB_RUNTIME/bin/backup
+## OR ##
+sudo $OPENHAB_RUNTIME/bin/backup /path/to/backups/folder/myBackup.zip
+```
+
+To restore from these generated files:
+
+```shell
+sudo $OPENHAB_RUNTIME/bin/restore $OPENHAB_BACKUPS/myBackup.zip
+```
+
+If you're unsure how to use the above files, just use `--help` or `-h`:
+
+```shell
+$OPENHAB_RUNTIME/bin/backup --help
+```
+
 ## Viewing Log Messages
 
 In order to get more insight on what your openHAB system is doing and to see occurring error messages, it is recommended to always have a look on the openHAB log files.
```

Comment:
Consider changing to, "You can learn more about openHAB and how it works by looking at your log files."

Reply:
👍 Might as well when I'm touching the file!